wiki/Installation.md

```diff
@@ -1,36 +1,70 @@
 # Swift Package Manager
-If you want more information about installing Swift packages with Xcode, [follow these instructions](https://developer.apple.com/documentation/swift_packages/adding_package_dependencies_to_your_app). If you want to learn more about Swift Package Manager, [this page goes into detail](https://swift.org/package-manager/).
+
+If you want more information about installing Swift packages through Xcode, [follow these instructions](https://developer.apple.com/documentation/swift_packages/adding_package_dependencies_to_your_app). If you want to learn more about Swift Package Manager, [this page goes into detail](https://swift.org/package-manager/).
+
+If you're a framework author and want to use Workflow as a dependency, update your `Package.swift` file:
 
 ## Get the package
+
 Add the following line to the package dependencies in `Package.swift`:
+
 ```swift
 .package(url: "https://github.com/wwt/Workflow.git", .upToNextMajor(from: "3.0.0")),
 ```
 
 ## Get the correct product
+
 Add one one of the following products to your target dependencies.
-### If you want to use Workflow with UIKit:
+
+#### **If you want to use Workflow with UIKit**
+
 ```swift
+.product(name: "Workflow", package: "Workflow"),
 .product(name: "WorkflowUIKit", package: "Workflow")
 ```
+
+#### **You'll import Workflow for UIKit as such**
```

Comment:
"as such" reads poorly, it's difficult to articulate why. I suggest alternate phrasing.

Reply:
Not going to duplicate this feedback but it should change in all places we used it.

Reply:
Phrasing updated to "Your import statement in this case will be" in [this commit](https://github.com/wwt/SwiftCurrent/pull/36/commits/37804d0febe7f73f07e54dde38bab04010030d58)